action.yml

```diff
@@ -123,20 +123,16 @@ runs:
         MANUAL_RUN_LIFETIME: ${{ inputs.MANUAL_RUN_LIFETIME }}
         OTHER_LIFETIME: ${{ inputs.OTHER_LIFETIME }}
       run: |
-        CURRENT_EPOCH=$(neofs-cli netmap epoch --rpc-endpoint $NEOFS_NETWORK_DOMAIN:8080)
         if [[ "${{ github.event_name }}" == "push" ]]; then
-          LIFETIME=$((MASTER_LIFETIME + CURRENT_EPOCH))
+          LIFETIME="$MASTER_LIFETIME"
```

Comment:
It depends on CLI flag used, old value is for `--expire-at`, this one is for `--lifetime`.

Reply:
> It depends on CLI flag used, old value is for `--expire-at`, this one is for `--lifetime`.

Yes, but gh-push-to-neofs uses the `--lifetime` flag:
https://github.com/nspcc-dev/gh-push-to-neofs/blob/master/action.yml#L99